closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -60,13 +62,17 @@ endif # OPENJCEPLUS_JGSKIT_PLATFORM
 
 compile-libs :
 	@$(ECHO) Compiling OpenJCEPlus native code
-	export \
-			$(OPENJCEPLUS_VS_LIB) \
-			GSKIT_HOME=$(OPENJCEPLUS_GSKIT_HOME) \
-			JAVA_HOME=$(OPENJCEPLUS_BOOT_JDK) \
-			JCE_CLASSPATH=$(OPENJCEPLUS_JCE_CLASSPATH) \
-			PLATFORM=$(OPENJCEPLUS_JGSKIT_PLATFORM) \
-		&& $(MAKE) -j1 -C $(OPENJCEPLUS_JGSKIT_MAKE_PATH) -f $(OPENJCEPLUS_JGSKIT_MAKE) all
+	$(EXPORT_COMPILER_ENV_VARS) \
+	$(MAKE) \
+		-C $(OPENJCEPLUS_JGSKIT_MAKE_PATH) \
+		-f $(OPENJCEPLUS_JGSKIT_MAKE) \
+		EXTERNAL_HEADERS=true \
+		GSKIT_HOME=$(OPENJCEPLUS_GSKIT_HOME) \
+		JAVA_HOME=$(OPENJCEPLUS_JDK) \
+		JCE_CLASSPATH=$(OPENJCEPLUS_JCE_CLASSPATH) \
+		OPENJCEPLUS_HEADER_FILES=$(OPENJCEPLUS_HEADER_FILES) \
+		PLATFORM=$(OPENJCEPLUS_JGSKIT_PLATFORM) \
+		all
```

Comment:
Suggest lines 66-75 be indented further to clarify that they are all part of the command that begins on line 65.

Reply:
Done.